src/utils.js

```diff
@@ -41,30 +41,37 @@ export function stripHash(regex) {
 
 /**
  * @param {number} delta
- * @param {number} difference
+ * @param {number} originalSize
  */
-export function getDeltaText(delta, difference) {
+export function getDeltaText(delta, originalSize) {
 	let deltaText = (delta > 0 ? '+' : '') + prettyBytes(delta);
-	if (delta && Math.abs(delta) > 1) {
-		deltaText += ` (${Math.abs(difference)}%)`;
+	if (originalSize === 0) {
+		deltaText += ` (new file)`;
+	} else if (originalSize === -delta) {
+		deltaText += ` (removed)`;
+	} else if (Math.abs(delta) > 0) {
+		const percentage = Math.round((delta / originalSize) * 100);
+		deltaText += ` (${delta > 0 ? '+' : '-'}${Math.abs(percentage)}%)`;
```

Comment:
can just let toString add the `-`:

```suggestion
		deltaText += ` (${delta > 0 ? '+' : ''}${percentage}%)`;
```